scipy/optimize/_linprog.py

```diff
@@ -187,18 +187,22 @@ def linprog(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None,
 
     Alternatively, that's:
 
-    minimize::
+        - minimize ::
```

Comment:
Technically I believe Sphinx interprets the indentation as a block quote. SciPy's theme no longer renders block quote text differently, but it used to show a vertical bar like

> This is a 
block quote

Note that the indentation is not needed if you only wanted to add the bullet.

Also, the bullet makes me think that is the following is a list. It's not really, and when linear programs are specified elsewhere (e.g. in papers and textbooks), I don't recall there being bullets.

Could you share the intent of the correction, or is it more a matter of personal preference?

Reply:
TLDR: personal preference.

I added the white space to avoid rendering the colon. For my personal taste, there were too many colons.
Then I added the bullet points to give it a bit more structure.

Reply:
If it were up to me, we'd just get rid of the code blocks altogether. IIRC, including both the mathematical and pseudocode version of the problem definition was a concession to those who don't like the way LaTeX looks in the rendered documentation. So maybe I'll let someone else review this part and merge.